src/applications/appeals/10182/components/AddIssue.jsx

```diff
@@ -185,6 +185,7 @@ const AddIssue = props => {
           onInput={handlers.onIssueNameChange}
           onBlur={handlers.onInputBlur}
           error={((submitted || inputDirty) && showIssueNameError) || null}
+          messageAriaDescribedby={content.hintText}
```

Comment:
should this be `content.name.hintText` like in 996?

Reply:
oops, yes!